kytos.json

```diff
@@ -3,7 +3,7 @@
   "username": "kytos",
   "name": "of_l2ls",
   "description": "A L2 learning switch application for OpenFlow switches.",
-  "version": "0.1.1",
+  "version": "0.3.0",
```

Comment:
Why 0.1.1 -> 0.3.0 ?

Should be 0.1.2, and if changes the REST API and/or is not compatible with previous version to 0.2.0

Reply:
My bad - it should be 0.2.0, it's a typo.